integreat_cms/static/src/js/media-management/component/upload-file.tsx

```diff
@@ -24,73 +27,48 @@ export default function UploadFile({
   mediaTranslations,
   submitForm,
   isLoading,
+  refreshListing,
 }: Props) {
-  // This state is a buffer which contains the currently selected file (the browser type File)
-  const [selectedFile, setSelectedFile] = useState<File | null>(null);
+  const dropZoneRef = useRef();
+
+  useEffect(() => {
+    const dropZone = new Dropzone(dropZoneRef.current, {
+      url: apiEndpoints.uploadFile,
+      headers: { "X-CSRFToken": getCsrfToken() },
+      dictDefaultMessage: mediaTranslations.text_upload_area,
+      acceptedFiles: allowedMediaTypes,
+      parallelUploads: 1,
+    });
+    dropZone.on("queuecomplete", () => {
+      setTimeout(() => {
+        refreshListing();
+        dropZone.removeAllFiles(false);
+      }, 500);
+    });
+    dropZone.on("complete", (file) => {
+      setTimeout(() => {
+        dropZone.removeFile(file);
+      }, 2000);
+    });
```

Comment:
```suggestion
```
I'm not sure what this event really does - if the action of the `queuecomplete` event is executed first, this will do nothing, right? If already all files have been removed, there is no need to remove the individual files as well...